modules/Makefile.circleci-2.0

```diff
@@ -78,3 +78,11 @@ circle\:run-job-kubernetes:
 	  echo "INFO: Running $(JOB_KUBERNETES_APP):$(IMAGE_TAG)"; \
 	  $(MAKEFILE_DIR)/bin/circle-do-exclusively.sh --branch $(CIRCLE_BRANCH) $(SELF) kubernetes:run-job kubernetes:job-logs KUBERNETES_APP=$(JOB_KUBERNETES_APP); \
 	fi
+
+## Install deployment dependencies
+install-deployment-dependencies:
```

Comment:
Using this approach of a mac and linux target would be beneficial as it allows testing of the makefile locally

https://github.com/sagansystems/cloud-platform/blob/8.0/conf/tf/Makefile#L14-L24